packages/flame_test/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   flutter_test:
     sdk: flutter
   meta: ^1.7.0
-  test: ^1.19.3
+  test: ^1.17.12
```

Comment:
Hmm, not sure about downgrading this, any reason for that?

Reply:
Ahhh, got it, this is conflicting with flutter's sdk test package.

We can revisit this later